docs/nodes/creating-nodes/node-review-checklist.md

```diff
@@ -0,0 +1,67 @@
+# Node review checklist
+
+If you want to create a new node for a service - that's great, thank you! There are several things to keep in mind when creating the node. To help you keep track of them, we prepared a checklist that covers the requirements for creating nodes, from preparation to submission. 
+
+Make sure you tick all the boxes below before submitting a node for review, as this will help our team review your PR easier and faster.
+
+## Preparation
+
+- [ ] Set up your editor for code formatting (indentation, new lines, linting). If you use Visual Studio Code, you can use the TSLint extension for linting.
+- [ ] Get credentials (e.g.: Client ID, Client Secret, API key, user login, user password, website URL) for the service you are building a node for.
+
+## Development
+
+- [ ] Ensure complementary operations (e.g. create, delete) have been added.
```

Comment:
Can we slightly elaborate this? Like maybe: complementary operation to each resource